lib/http/http.c

```diff
@@ -506,6 +507,20 @@ gotheaders(struct http_cookie * H, uint8_t * buf, size_t buflen)
 		H->res.headers[i].header = s;
 		H->res.headers[i].value = s[cpos] ? &s[cpos + 1] : &s[cpos];
 		s[cpos] = '\0';
+
+		/* Strip leading optional whitespace (OWS) from the value. */
+		j = 0;
+		while (H->res.headers[i].value[j] == ' '
```

Comment:
This seems like a very long-winded way of writing `H->res.headers[i].value += strspn(H->res.headers[i].value, " \t");`

Reply:
nice, done